src/lib/utilities/strings.ts

```diff
@@ -254,3 +254,11 @@ export function escapeTsQueryCharactersAndFormatPrismaSearch(text: string): stri
     return undefined;
   }
 }
+
+export function sortUuids(uuids: string[]): string[] {
+  return uuids.sort((a, b) => a.localeCompare(b));
+}
+
+export function extractUuids<T extends { id: string }>(items: T[]): string[] {
```

Comment:
i'd honestly rather just see the map() inline in the code then a function that does this